tests/unit/controllers/pull-requests-test.js

```diff
@@ -158,12 +158,61 @@ module('Unit | Controller | pull-requests', function(hooks) {
 
     assert.strictEqual(
       controller.contributorsList,
-      [
-        '<a href="https://github.com/emberjs" rel="noopener noreferrer" target="_blank">@emberjs</a>',
-        '<a href="https://github.com/tomster" rel="noopener noreferrer" target="_blank">@tomster</a>',
-        '<a href="https://github.com/zoey" rel="noopener noreferrer" target="_blank">Zoey (@zoey)</a>'
-      ].join(', '),
+        '<a href="https://github.com/emberjs" rel="noopener noreferrer" target="_blank">@emberjs</a>, <a href="https://github.com/tomster" rel="noopener noreferrer" target="_blank">@tomster</a>, and <a href="https://github.com/zoey" rel="noopener noreferrer" target="_blank">Zoey (@zoey)</a>'
+      ,
       'We can update the contributors list.'
     );
   });
+
+  test('addOxfordComma works', function (assert) {
+    const controller = this.owner.lookup('controller:pull-requests');
+
+    controller.updateContributorsList([]);
+    assert.strictEqual(
+      controller.contributorsList,
+        ''
+      ,
+      'We return an empty string for no contributors.'
+    );
+
```

Comment:
Can we add 1 more assertion to test the case of 1 contributor?

Reply:
Yes good call.